examples/jsm/shaders/FXAAShader.js

```diff
@@ -156,46 +156,46 @@ const FXAAShader = {
 
 			// 45 deg edge detection and corners of objects, aka V/H contrast is too similar
 			if( abs( relativeVContrast ) < .3 ) {
```

Comment:
This comparison isn't very intuitive, I just changed tab...

Reply:
I wouldn't change the code at all since it is from a reference implementation.